.github/workflows/ci-aarch64-native.yml

```diff
@@ -31,7 +31,7 @@
 
 # Github Actions workflow for aarch64 Continuous Integration testing.
 
-name: ci-aarchxx
+name: ci-aarch64-native
```

Comment:
Note that this will break the required test list which is by name so that will need an admin update (normally after the 1st one is run; maybe the old one is still there though).

Reply:
That's what I figured. I have email outstanding to ARM folks asking exactly that (as in "what did they do when they changed the name").

Reply:
I can update the required list if they're ok with the change, but please send this to @joshua-warburton for review: maybe there was a reason for the name change we are not aware of.

Reply:
It was renamed to be more consistent with the existing tests which are generally ci-[archname]/agent

Reply:
I could see it either way on the name: I don't have strong feelings.  The one thing I would like to improve if it's easy is to speed up the ubuntu-20-arm64-sve which is taking 20 minutes and is the slowest of our tests.

Reply:
Can we change name of the other ci-aarchxx workflow then?